framework/doc/content/source/mfem/auxkernels/MFEMComplexExteriorProductAux.md

```diff
@@ -0,0 +1,25 @@
+# MFEMComplexExteriorProductAux
```

Comment:
This file is based on an old version of `MFEMInnerProductAux.md`, the new version improves the language and tex notation, can we use that instead?